rpm/rust-cbindgen.spec

```
@@ -0,0 +1,70 @@
+%global rustflags -Clink-arg=-Wl,-z,relro,-z,now
+
+Name:           cbindgen
+Version:        0.16.0
+Release:        0
+Summary:        A tool for generating C bindings from Rust code
+License:        MPL-2.0
+URL:            https://crates.io/crates/cbindgen
+Source:         %{name}-%{version}.tar.bz2
+BuildRequires:  cargo >= 1.30.0
+BuildRequires:  rust >= 1.30.0
+BuildRequires:  rust-std-static >= 1.30.0
+
+%description
+A tool for generating C bindings from Rust code.
+
+%prep
+%autosetup -n %{name}-%{version_no_tilde}/cbindgen -p1
```

Minor cosmetic proposal:
```suggestion
%autosetup -p1 -n %{name}-%{version}/%{name}
```